pkg/workload/workload.go

```diff
@@ -520,3 +524,40 @@ func RemoveFinalizer(ctx context.Context, c client.Client, wl *kueue.Workload) e
 	}
 	return nil
 }
+
+// AdmissionChecksForWorkload returns AdmissionChecks that should be assigned to a specific Workload based on
+// ClusterQueue configuration and ResourceFlavors
+func AdmissionChecksForWorkload(log *logr.Logger, wl *kueue.Workload, cq *kueue.ClusterQueue) sets.Set[string] {
```

Comment:
Logger is already an interface


```suggestion
func AdmissionChecksForWorkload(log logr.Logger, wl *kueue.Workload, cq *kueue.ClusterQueue) sets.Set[string] {
```

Reply:
Fixed